Shops/ShopGoblinTinkerer.cs

```diff
@@ -134,234 +118,216 @@ public void InitShop(string currentShop)
-                            shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                         }
                         shop.item[nextSlot].SetDefaults(ItemID.HandWarmer);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                         if (Main.hardMode)
                         {
                             shop.item[nextSlot].SetDefaults(ItemID.ObsidianRose);
-                            shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                         }
 
                         if (Utils.kills(NPCID.Medusa) > 0)
                         {
                             shop.item[nextSlot].SetDefaults(ItemID.PocketMirror);
-                            shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                         }
 
 
                         if (Utils.kills(NPCID.LightMummy) > 0)
                         {
                             shop.item[nextSlot].SetDefaults(ItemID.TrifoldMap);
-                            shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                         }
 
                         if (Utils.kills(NPCID.Corruptor) > 0)
                         {
                             shop.item[nextSlot].SetDefaults(ItemID.Vitamins);
-                            shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                         }
                         if (Utils.kills(NPCID.CursedSkull) > 0)
                         {
                             shop.item[nextSlot].SetDefaults(ItemID.Nazar);
-                            shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                         }
                     }
                     break;
                 case "Combat":
                     if (NPC.downedBoss2)
                     {
                         shop.item[nextSlot].SetDefaults(ItemID.PanicNecklace);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                     }
                     if (NPC.downedPlantBoss)
                     {
                         shop.item[nextSlot].SetDefaults(ItemID.BlackBelt);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                     }
                     if (NPC.downedQueenBee)
                     {
                         shop.item[nextSlot].SetDefaults(ItemID.HoneyComb);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                     }
 
                     if (NPC.downedMechBoss1)
                     {
                         shop.item[nextSlot].SetDefaults(ItemID.MoonCharm);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
```

Comment:
You need `++` otherwise everything will be merged into one slot.

Please review the prices.